### DISCUSSION TITLE
docs/concepts/evaluation

### DISCUSSION THREAD

**Discussion by giscus** [gorse-io/docs#70] [Giscus]
# docs/concepts/evaluation

Evaluation To estimate the performance of a recommender system, evaluation is needed. Gorse provides both online evaluation and offline evaluation. Online Evaluation The target of a recommender system is to maximize the probability every user like recommended items. Thus, the metric for online evaluation is the positive feedback rate

https://gorse.io/docs/concepts/evaluation.html

**Comment by agtabesh** [comment]
Using `recall` only, as a metric, for evaluation is not useful because you could recommend the whole items so that `recall` would be equal to `1`. so `recall` and `precision` simultaneously could be informative. Considering `f1-score`, which is a combination of `recall` and `precision`, could be far better to be used in recommendation engines.
